### PR TITLE
[mediaqueries-5] Migrate Web Preferences API proposal

### PR DESCRIPTION
**Pull request by lukewarlow**
Per resolution at https://github.com/w3c/csswg-drafts/issues/6517#issuecomment-2220975121 this migrates the [Web Preferences API](https://wicg.github.io/web-preferences-api/) to the Media Queries Level 5 spec.

### CODE REVIEW

File: mediaqueries-5/Overview.bs
Comment (lukewarlow) [inline mediaqueries-5/Overview.bs:3223]: Probably needs a bit of a better introduction here. Perhaps with more from https://wicg.github.io/web-preferences-api/#sec-intro

**Comment by lukewarlow** [comment]
TODO: It seems like the proposal and the spec use different flavours of bikeshed syntax, so I probably need to migrate to match the spec @tabatkins I don't suppose there's a tool to convert from the shorthand to the more HTML like syntax is there?

TODO: Add to the privacy section, at the very least an inline issue to signpost that it needs filling in.

**Comment by tabatkins** [comment]
> so I probably need to migrate to match the spec

Don't worry about it.

> Add to the privacy section, at the very least an inline issue to signpost that it needs filling in.

Doesn't need much since there isn't *actually* a privacy concern, but it would be good to highlight *why* there isn't, yes, along the lines of what we've discussed in the issues. Maybe something like:

> The {{PreferenceManager}} object allows querying some user-preference [=media features=]. This is not a privacy leak, as that information was always trivially available by using the [=media features=] themselves in the normal fashion. 
>
> The {{PreferenceManager}} object also allows overriding these user-preference [=media features=]; this is also neither a privacy nor accessibility regression, as the [=media features=] were already trivially ignorable by simply not querying them.

**Comment by lukewarlow** [comment]
@tabatkins apologies for the massive delay in getting back to this. I've added in your suggestions to the privacy section. I'd appreciate an overall review when you get time. I'll work on migrating the issues across to this repo, probably once this PR is merged.

**Review by tabatkins — APPROVED** [review]

**Comment by lukewarlow** [comment]
@tabatkins is this ready to merge? Just an fyi I don't have merge rights so I can't do it myself :)

**Comment by lukewarlow** [comment]
Thanks!

**Pull request merged**
